.github/workflows/publish.yaml

```diff
@@ -0,0 +1,44 @@
+name: Image Checks
+
+on:
+  push:
+    # Run only on release pushes
+    tags:
+      - v*
+  # but all PRs
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build image
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          cache-from: type=gha
+          cache-to: type=gha
+          tags: ci_test
```

Comment:
load: true 

here, instead of cache-from in your Publish Image might be a quicker option. You would also want this if you are going to run some tests in the container at some point.

Reply:
Thanks